lib/assesments/assessment_2/fi-as2-hira/Screens/my_profile/widgets/fi_as2_hira_text.dart

```diff
@@ -0,0 +1,31 @@
+part of '../fi_as2_hira_profile.dart';
+
+class H1Text extends StatelessWidget {
+  final String text1;
+  final String text2;
```

Comment:
instead of text1 name something more descriptive like heading and subheading etc